### PR TITLE
feat: [refactor]move CollectorController to CollectorService

### PR DESCRIPTION
**Pull request by MRgenial**
I have moved the collectorController to collectorService.

### CODE REVIEW

File: manager/src/main/resources/application.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ spring:
   application:
     name: ${HOSTNAME:@hertzbeat@}${PID}
   profiles:
-    active: prod
+    active: dev
```
Comment (yuluo-yx) [inline manager/src/main/resources/application.yml:21]: Why change the configuration file here?

**Review by yuluo-yx — COMMENTED** [review]
imo, the modification of a pr should be as single as possible to facilitate review, here the modification of the configuration file and the controller code movement is not synchronized. Thank you for contributing to this!

**Comment by yuluo-yx** [comment]
I see this is a duplicate of #2433 🥲

**Comment by MRgenial** [comment]
> I see this is a duplicate of #2433 🥲

oh,sorry,i didnt see it.

**Comment by yuluo-yx** [comment]
@2433 merged, you can close it.

**Pull request closed without merging**